packages/flutter/test/material/input_decorator_test.dart

```diff
@@ -5399,9 +5396,9 @@ void main() {
           // Right padding is 12 per Material 3 spec.
           expect(getDecoratorRect(tester).right - tester.getRect(findSuffixIconInnerRichText()).right, 12.0);
           // Check the padding between the suffix icon and the input.
-          // TODO(bleroux): the gap between the icon and the input should be 16 based on M3 specification.
+          // The gap between the icon and the input should be 16 based on M3 specification.
           // See https://github.com/flutter/flutter/issues/149409.
```

Comment:
Remove this reference to the issue, this link was required for the TODO (it no more required now).